assets/js/modules/analytics/datastore/__fixtures__/select-property-property-profiles.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Rather than create a new fixture variant here, I think we can use the existing `properties-profiles.json` and simply update the `defaultPropertyId` to be a non-existent profile ID (e.g. `99999`). We only need to do this on a copy of the fixture to ensure we're not interfering with other tests that use it.
e.g.
```js
const nonExistentProfileID = "99999";
const propertiesProfiles = {
	...fixtures.propertiesProfiles,
	properties: fixtures.propertiesProfiles.properties.map(
		( property ) => { ...property, defaultProfileId: nonExistentProfileID }
	)
};
// ...
expect( getProfiles().some( ( { id } ) => id === nonExistentProfileID ) ).toBe( false );
expect( getProfileID() ).not.toBe( nonExistentProfileID )
```